app/assets/stylesheets/application.scss

```diff
@@ -1,9 +1,3 @@
-$govuk-assets-path: "/agent-services-account/assets/lib/govuk-frontend/govuk/assets/";
```

Comment:
Everything else looks good. Why is this removed? Is this because of the issues with local styles you were having... Is it not fixed if we are using `hmrcLayout` ?

Reply:
yes. don't need this now

Reply:
Okay